### DISCUSSION TITLE
100k dataset test label file?

### DISCUSSION THREAD

**Discussion by sainisanjay** [bdd100k/bdd100k#52] [Q&A] [answered]
100k dataset is divided into train(70k), test(20k) and val(10k) but we can see `Detection 2020 Labels` only provide train (`det_v2_train_release.json`) and validation (`det_v2_val_release.json`) labels file. Where we can fine test label file?

**Comment by XiaLiPKU** [comment]
We will not publish the test label file. However, we are preparing the evaluation server, which shall start to work this month. You can test it on that server then

**Reply by Abdul-Nasir11** [reply]
Hello @XiaLiPKU any update regarding the evaluation server, is it live?

**Reply by NiranjanRavi1993** [reply]
@Abdul-Nasir11  Hi, did you manage to acquire the test labels?

**Comment by fyu** [comment, marked as answer]
You can find all the evaluation server links here https://www.bdd100k.com/#leaderboards
